main.js

```diff
@@ -1 +1,37 @@
 import customers from './customers.js'
+
+//Info needed: Name, address, dob, email, customer since, thumbnail/pic
+// let person = { 
+//   name: "Christine Cooper"
+//   // thumbnail: 'picture': {
+//   //     'large': 'https://randomuser.me/api/portraits/men/54.jpg',
+//   //     'medium': 'https://randomuser.me/api/portraits/med/men/54.jpg',
+//   //     'thumbnail': 'https://randomuser.me/api/portraits/thumb/men/54.jpg'
+//   address: "7725 Fincher Rd., Utica, WV 34757"
+//   dob: "Feb 16, 1974"
+//   email: "christinecooper@example.com"
+//   customer since: "Sep 25, 2007"
+
+console.log(customers[2])
+let customer1=customers[2];
+console.log(customer1.name[2])
+
+// }
+
+// function renderCustomers (personarray) {
+ 
+// // Create element or use an existing one
+// let ELEMENT = 
+// document.getElementById("customerlist");
+//   // document.getElementById()
```

Comment:
You defined the variable ELEMENT in 24, so that's what you'll have to use. You can set the innerHTML, but you probably are better off creating elements using document.createElement and then appending whatever you need to those elements.